x-pack/plugins/synthetics/public/apps/synthetics/state/settings/api.ts

```diff
@@ -61,6 +61,7 @@ export const fetchConnectors = async (): Promise<ActionConnector[]> => {
       is_preconfigured: isPreconfigured,
       is_deprecated: isDeprecated,
       is_missing_secrets: isMissingSecrets,
+      is_system_action: isSystemAction,
```

Comment:
is it possible to move this util to actions plugin? or reuse?

Reply:
I think it is better if the action plugin exposes a function that does the API call and the transformation as we do in [cases]([x-pack/plugins/synthetics/public/apps/synthetics/state/settings/api.ts](https://github.com/elastic/kibana/pull/160753/files/ee5df7b9c8b7561b5e831a1c318845c454d0e9f0#diff-c473d35bf008584722f7ef58add7e9b4eb293068ec1ff4c000da5b0825baf075)). I think the recommendation is to not call the APIs of other plugins directly but use a thin client. This way the team can do changes in the API without affecting other teams. This will also be helpful when versioning our endpoints. @pmuellr Is there any issue for that?

Reply:
Ok, I will open an issue. @shahzad31 What do you think of this approach (addressing it on another PR)?

Reply:
Done https://github.com/elastic/kibana/issues/new